scripts/get_3rdparty_jars.py

```diff
@@ -0,0 +1,8 @@
+import subprocess
+
+output = subprocess.check_output("./pants dependencies : 3rdparty/intellij:: | sort | grep 'com\|org\|jdk'", shell=True)
```

Comment:
Interesting. I wouldn't have expected `:` by itself to be a valid commandline address.